src/components/CanvasSaveButton/CanvasSaveButton.scss

```diff
@@ -0,0 +1,22 @@
+.tm-canvas-save-button,
+.tm-canvas-save-button:hover {
+  background-color: #666;
+  border-radius: 50%;
+  cursor: pointer;
+  font-size: 30px;
+  margin: 0 10px;
+  text-align: center;
+  transition: background-color 0.5s ease, color 0.5s ease;
+  width: 35px;
+  height: 35px;
+  color: white;
+
+  &:hover {
+    background-color: $btn-primary-color-hover;
+  }
+
+  svg {
+    padding-top: 3px;
+    width: 75%;
+  }
+}
```

Comment:
Using the className property in ShareMenu example you can reduce significantly the number of css for tm-canvas-save-button. And if there is still css left, you can probably move it to CanvasSaveButton.md.scss.

We must not forget that a button will often be used in a parent and this is the parent that will define his css.